selfdrive/controls/radard.py

```diff
@@ -11,7 +11,7 @@
 from selfdrive.controls.lib.cluster.fastcluster_py import cluster_points_centroid
 from selfdrive.controls.lib.radar_helpers import Cluster, Track, RADAR_TO_CAMERA
 from system.swaglog import cloudlog
-
+from selfdrive.car.toyota.values import RADAR_ACC_CAR_TSSP
```

Comment:
your car-specific code shouldn't touch radard like this

Reply:
ok, then we have to update RadarPoint to support absolute speed.

Reply:
@sshane Shall we add a union to RadarPoint or simple add a field named v and set vRel to nan when we want to use v instead?

Reply:
You could pass in vEgo to RI.update() and set vRel appropriately with that information. Or add v to RadarPoint in cereal and have radar use whichever is set (or add a bool). I'd try the first option since that will be easier without any cereal changes